src/ContactPage/Contact.scss

```diff
@@ -0,0 +1,68 @@
+@import "../styles/typography";
+
+.contact-section {
+  height: 100vh;
+
+  &__border {
+    border: none;
+    padding: 2% 0;
+    background: rgb(229, 226, 226);
+  }
+  &__title {
+    font-family: $base-font-family;
+    margin: 0 5%;
+    margin-bottom: 10px;
+  }
+
+  &__form {
+    margin: 0 5%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__submit {
+    background-color: rgb(238, 59, 74);
+    border-radius: 5px;
+    color: white;
+    border: none;
+    text-align: center;
+    text-decoration: none;
+    font-size: 1em;
+    width: 120px;
+    height: 30px;
+    margin-top: 10px;
+
+    &:disabled {
+      background-color: grey;
+    }
+  }
+}
+
+input[type="text"],
+input[type="email"],
+.contact-section__message {
```

Comment:
Why not give all these elements the same class name if they've all got the same styling? Also classname selectors are better than element selectors so always try to use those when you can